packages/node-sdk/paima-utils-backend/src/cde-access.ts

```diff
@@ -15,8 +15,18 @@ import {
   internalGetCardanoAddressDelegation,
   internalGetCardanoProjectedNft,
   internalGetCardanoAssetUtxos,
+  internalGetErc1155TotalBalanceAllTokens,
+  internalGetErc1155AllTokens,
+  internalGetErc1155ByTokenId,
+  internalGetErc1155ByTokenIdAndWallet,
 } from './cde-access-internals.js';
-import type { ICdeCardanoGetProjectedNftResult } from '@paima/db/src';
+import type {
+  ICdeCardanoGetProjectedNftResult,
+  ICdeErc1155GetAllTokensResult,
+  ICdeErc1155GetByTokenIdAndWalletResult,
+  ICdeErc1155GetByTokenIdResult,
+} from '@paima/db/src';
```

Comment:
don't import from `src` directory. If you import `src` folders TS is not smart enough to know you're trying to leverage the build output of the `@paima/db` compilation unit, and it will instead inline the src folders part of this compilation unit instead.